### PR TITLE
QTM 453 - [SSR-SIMULATOR] Implement new HTML

### PR DESCRIPTION
**Pull request by alizeleal**
https://jirasoftware.catho.com.br/browse/QTM-453

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,16 +1,91 @@
 import App from 'next/app';
 import Head from 'next/head';
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (rapahaeru) [inline pages/_app.js]: is this beeing used?

File: pages/_app.js [hunk 2]
```diff
@@ -1,16 +1,91 @@
...
 import { GlobalStyle } from '@catho/quantum';
+import Image from 'next/image'
+import logoImage from '../static/imgs/cathoLogo.png'
+import backgroundImg from '../static/imgs/background-image.png'
```
Comment (rapahaeru) [inline pages/_app.js]: what is the pattern for the image names?
one is img another os Image. 
How about of choose only one patter?

File: pages/_app.js [hunk 3]
```diff
@@ -1,16 +1,91 @@
 import App from 'next/app';
 import Head from 'next/head';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { GlobalStyle } from '@catho/quantum';
+import Image from 'next/image'
```
Comment (rapahaeru) [inline pages/_app.js]: Doubt: why is this necessary? a tag img from html is not sufficient?

File: pages/_app.js [hunk 4]
```diff
@@ -1,16 +1,91 @@
...
+import logoImage from '../static/imgs/cathoLogo.png'
+import backgroundImg from '../static/imgs/background-image.png'
+
+
 
 export default class HomePage extends App {
+
+  componentDidMount() {
+
```
Comment (rapahaeru) [inline pages/_app.js]: why all this spaces? How about of keep the code closer? 
Just other codes here..

File: pages/_app.js [hunk 5]
```diff
@@ -1,16 +1,91 @@
...
+      </div>
+      <nav id="hiddenMenu" class="HiddenMenu HideMenu">
+        <ul>
+          <li class="navItem"><a href="#">Contact us</a></li>
+          <li class="navItem"><a href="#">Repositories</a></li>
+        </ul>
+      </nav>
+    </header>
+    <div  class="MainContent">
         <GlobalStyle />
```
Comment (rapahaeru) [inline pages/_app.js]: How about of move the GlobalStyle to the top of html?
The globalstyle is responsible to keep the pattern of the CSS and its fonts..

File: pages/_document.js [hunk 6]
```diff
@@ -32,17 +32,27 @@ export default class MyDocument extends Document {
...
           />
           {this.props.styleTags}
         </Head>
+
         <body>
-          <Main />
-          <NextScript />
+
+        <main>
+
```
Comment (rapahaeru) [inline pages/_document.js]: how about of remove this spaces?

File: pages/function.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (rapahaeru) [inline pages/function.js]: is this file necessary?

File: pages/index.js [hunk 8]
```diff
@@ -9,6 +9,7 @@ const colsProps = {
 
 export default () => (
   <>
+
```
Comment (rapahaeru) [inline pages/index.js:12]: how about of remove this spaces?

File: static/style.css [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+
+
+.navItem {
+    float:left;
+    margin-right: 136px;
+    display: flex;
+}
+
+.NavList {
+
```
Comment (rapahaeru) [inline static/style.css:30]: how about of this identation be corrected?

File: static/style.css [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+.HeaderContainer{
+    display: flex;
+    justify-content: space-between;
+    height: 77px;
+    padding: 24px 72px;
+}
+
+.Footer {
+    min-height: 66px;
+    font-family: "Nunito", sans-serif;
```
Comment (rapahaeru) [inline static/style.css]: the global style imports the Nunito font family, is this declaration necessary?

File: static/style.css [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.HiddenMenu li{
+    float: right;
+}
+
+.HiddenMenu a{
+
```
Comment (rapahaeru) [inline static/style.css]: spaces

File: static_files/index.html [hunk 12]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
```
Comment (rapahaeru) [inline static_files/index.html]: is this file necessary?

**Review by rapahaeru — CHANGES_REQUESTED** [review]

File: pages/_app.js (hunk 1 above)
Comment (alizeleal) [reply pages/_app.js]: Removed

File: pages/_app.js (hunk 2 above)
Comment (alizeleal) [reply pages/_app.js]: The second isn't being used, I happened to forget to remove.

File: pages/_app.js (hunk 3 above)
Comment (alizeleal) [reply pages/_app.js]: The image has different behaviors when called from the img tag and when using this component.
With this component, the size of the image changes according to the size of the screen. I had trouble showing the logo and found this solution using the Image component. However, for the purpose of this project and after changing the folders, it now works with

File: static_files/index.html (hunk 12 above)
Comment (alizeleal) [reply static_files/index.html]: Leftovers from the previous versions. Removed.

File: static/style.css (hunk 10 above)
Comment (alizeleal) [reply static/style.css]: Removed since it is redundant now

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code review
[x] Build
[x] Layout

**Review by rapahaeru — APPROVED** [review]
